packages/playwright/src/reporter.ts

```diff
@@ -24,6 +24,7 @@ import { getPlaywrightBrowserPath } from "@replayio/replay";
 import { FixtureStepStart, ParsedErrorFrame, TestIdData } from "./fixture";
 import { StackFrame } from "./playwrightTypes";
 import { getServerPort, startServer } from "./server";
+import { telemetryLogger } from "@replayio/telemetry-node";
```

Comment:
if we end up using it here, we should add a reference to this package in https://github.com/replayio/replay-cli/blob/7cc25e7717dc8a1bb21b4b3df99f46cafeddc192/packages/playwright/tsconfig.json

The time will probably come soon that I'll have to automate generating those references 😅

Reply:
Got it.